traefik/tests/test_check.py

```diff
@@ -0,0 +1,96 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+import subprocess
+import os
+import time
+import socket
+import requests
+
+from datadog_checks.utils.common import get_docker_hostname
+from datadog_checks.traefik import TraefikCheck
+from datadog_checks.errors import CheckException
+
+
+def test_check(aggregator):
+
+    c = TraefikCheck('traefik', {}, {}, None)
+
+    c.check({'host': 'foobar'})
+    aggregator.assert_all_metrics_covered()
+
+    instance = {}
+    with pytest.raises(CheckException):
+        c.check(instance)
+
+    with pytest.raises(CheckException):
+        c.check({'port': '16080'})
+
+    with pytest.raises(CheckException):
+        c.check({'path': 'foo'})
+
+
+@pytest.mark.integration
+def test_service_check(aggregator):
+    c = TraefikCheck('traefik', {}, {}, None)
+
+    HERE = os.path.dirname(os.path.abspath(__file__))
+    args = [
+        "docker-compose",
+        "-f", os.path.join(HERE, 'docker-compose.yml')
+    ]
+
+    # start the Traefik container
+    subprocess.check_call(args + ["up", "-d"])
```

Comment:
Can you please makes fixtures in `conftest.py` for spinning up the required containers? Here are a few examples:

https://github.com/DataDog/integrations-core/blob/master/envoy/tests/conftest.py
https://github.com/DataDog/integrations-core/blob/master/openldap/tests/conftest.py

And here are some docs 🙂 https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/docker.py#L46

Reply:
done.